sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateOctKeyOptions.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.keyvault.keys.models;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+public class CreateOctKeyOptions extends CreateKeyOptions {
+    /**
+     * The hardware protected indicator for the key.
+     */
+    private boolean hardwareProtected;
+
+    /**
+     * Creates a CreateOctKeyOptions with {@code name} as name of the Rsa key.
```

Comment:
```suggestion
     * Creates a CreateOctKeyOptions with {@code name} as name of the AES key.
```

Also, capitalize RSA (it's an acronym) in its source file.